kotlin/internal/jvm/android.bzl

```diff
@@ -57,6 +59,7 @@ def _kt_android_artifact(
         visibility = ["//visibility:private"],
         kotlinc_opts = kotlinc_opts,
         javac_opts = javac_opts,
+        tags = tags,
```

Comment:
Not passing the tags here prevents people from conditionally disabling Kotlin ABI compilation via the `tags` attribute